src/index.jsx

```diff
@@ -135,25 +135,35 @@ let Ink = React.createClass({
   },
 
   render() {
-    let { density, height, width, touchEvents } = this.state
+    const { density, height, width, touchEvents } = this.state
 
     return (
-      <canvas className="ink"
+      <canvas className={this.props.className || "ink"}
               style={{ ...STYLE, ...this.props.style }}
               height={ height * density }
               width={ width * density }
               onDragOver={ this._onRelease }
+              { ...this._filterProps()}
               { ...touchEvents } />
     )
   },
 
+  _filterProps () {
+    return Object.keys(this.props).reduce((filtered, key) => {
+      if (key !== 'style' && key !== 'className' && key !== 'onDragOver') {
+        filtered[key] = this.props[key]
+      }
+      return filtered
+    }, {})
+  },
```

Comment:
This whole block (of my doing) feels a bit gnarly. I'm going to make a note to revisit this (#9)